clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/provider/agent/KubernetesConfigMapCachingAgent.groovy

```diff
@@ -30,17 +33,17 @@ import io.fabric8.kubernetes.api.model.ConfigMap
 import static com.netflix.spinnaker.cats.agent.AgentDataType.Authority.AUTHORITATIVE
 
 @Slf4j
-class KubernetesConfigMapCachingAgent extends KubernetesCachingAgent {
+class KubernetesConfigMapCachingAgent extends KubernetesCachingAgent<KubernetesV1Credentials> {
```

Comment:
How likely is it that a caching agent will need to support > 1 version?

Reply:
Not very, since each caching agent is coupled to a single account, and each account has one credential of a single version.